website_sale_google_shopping/models/product.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# License, author and contributors information in:
+# __openerp__.py file at the root folder of this module.
+from openerp import models, fields
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    google_product_category_ids = fields.Many2many(
+        string='Categories',
+        comodel_name='google_product_category',
+        relation='product_tmpl2google_product_category',
+        column1='product_tmpl_id',
+        column2='google_product_category_id')
+
+    google_color_id = fields.Many2one(
```

Comment:
IMO this is a lot of model attributes to add to the namespace. I think we should instead utilize the pre-existing Product Attributes relation by adding a `google_attribute_id` to relate to the specific attribute.